web/annotation_editor_layer_builder.css

```
@@ -309,11 +309,12 @@
       height: 100%;
 
       .divider {
-        width: 1px;
+        width: 0;
         height: calc(
           2 * var(--editor-toolbar-padding) + var(--editor-toolbar-height)
         );
-        background-color: var(--editor-toolbar-border-color);
+        border-top: 1px solid var(--editor-toolbar-border-color);
+        border-bottom: none;
```

This seems to remove the separator shown for e.g. the Stamp-editor inline toolbar, surely this should be using left/right instead?
```suggestion
        border-left: 1px solid var(--editor-toolbar-border-color);
        border-right: none;
```